### DISCUSSION TITLE
Option to use percentage number instead of battery icon

### DISCUSSION THREAD

**Discussion by HubKing** [valinet/ExplorerPatcher#1044] [Ideas]
When I first enabled Windows 10-style taskbar, clicking the battery icon did nothing. It just showed the hourglass briefely and showed nothing. I ran the settings and changed "System tray -> Battery' to "Windows 7", and changed it back to "Windows 10 flyout". Then it showed the Windows-10-style popup when I clicked the battery icon.

Anyway, if the Windows-10-style task bar is not what Windows creates but what Explorer Patcher creates to mimic Windows-10's taskbar, how about having an option to simply show the **battery percentage number instead of the battery icon**? I always prefer a number instead of the battery icon, for which it is hard to know what exact the value is. Other operating systems like Ubuntu/Android have an option to display the percentage number, but Windows, for some reason, does not.

![image](https://user-images.githubusercontent.com/10900489/157573559-cfd29957-f815-44f0-838d-5766eeb464b0.png)

**Comment by valinet** [comment]
No, the Windows 10 taskbar is actually from explorer, and ExplorerPatcher does quite some stuff to fix it and make it fully functional (that’s why it was named “patcher” originally), but it is not a reimplementation from scratch, as Simple Window Switcher is for Alt-Tab, for example.

That being said, with the way the system tray works now, not really an option. Anyway, you could use [Battery Util](https://github.com/tarcode-apps/BatteryMode) which is a much much better alternative to Windows’ flyout; basically obliterating the need for it. I highly recommend it. I will check to make sure, but it has plenty of options for its icon, and I think it has one for percentages too.

**Reply by HubKing** [reply]
I have tried the utility you have introduced, and it does have the percentage display mode. I think I will use this for that purpose; thanks for the introduction. 

By the way, I am not sure if this is because my laptop is slow or not but, if I use "Windows 10 flyout" for the battery icon, there is about 1-second delay until it appears after I clicked the icon. If I set it to "Windows 7", then the pop-up is instant.
